docs/source/tools/trigger-service/authorization.rst

```diff
@@ -0,0 +1,157 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
It would be nice to point to a full example setup of this but let’s leave this out of this PR.